.github/workflows/test-check-dependent-jobs.yml

```diff
@@ -0,0 +1,42 @@
+name: test-check-dependent-jobs
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/test-check-dependent-jobs.yml'
+      - 'check-dependent-jobs/**'
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/test-check-dependent-jobs.yml'
+      - 'check-dependent-jobs/**'
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./check-dependent-jobs
+        id: check
+        with:
+          needs: '{ "job-a": { "result": "success" }, "job-b": { "result": "success" } }'
+      - name: assert
```

Comment:
**nit**: maybe the expected value in the name of the step?

Reply:
https://github.com/elastic/oblt-actions/pull/16/commits/059a02bc851bbb029a31c59ab31184948683aabe